src/hooks/useInteractive.ts

```diff
@@ -7,6 +7,7 @@ import {
   getIsHovered,
 } from '../core/selectors/components'
 import { getShowLayout, getFocusedComponent } from '../core/selectors/app'
+import { useDropComponent } from './useDropComponent'
```

Comment:
not sure if this is a good idea. useInteractive is a custom hook which makes a component draggable. Not sure if adding useDropComponent hook here is a good idea

Reply:
As useInteractive is a common hook , I used there to reduce the lines of code and also we need ref to know the boundary points of the component. As we are creating ref in the useInteractive. These are the two reasons why i used useDropComponent in useInteractive. I will find a way to use it outside bro